package.json

```diff
@@ -8,10 +8,12 @@
   "scripts": {
     "release": "changeset publish",
     "build:docs": "pnpm turbo build --filter docs-app",
-    "start": "pnpm start:tests",
-    "start:docs": "pnpm turbo --filter docs-app start",
-    "start:tests": "pnpm turbo --filter test-app start",
-    "start:addon": "pnpm --filter !test-app start --no-watch.clearScreen",
+    "start": "concurrently 'npm:start:*(!only)' --names 'start:'",
+    "start:docs": "pnpm --filter docs-app start",
+    "start:tests": "pnpm --filter test-app start",
+    "start:addon": "pnpm --filter './packages/*' start --no-watch.clearScreen",
+    "start:only:docs": "pnpm --stream --parallel --filter docs-app... start",
```

Comment:
This `...` syntax is pretty cool... It's part of `pnpm`'s [filtering syntax](https://pnpm.io/filtering) and means "automatically run `start` for dependencies of `docs-app` first".

In a way it could potentially remove the need for `turbo` but turbo does give caching as well...